lib/std/zon/stringify.zig

```diff
@@ -0,0 +1,2088 @@
+            /// `valueArbitraryDepth`.
+            pub fn fieldArbitraryDepth(
+                self: *Struct,
+                name: []const u8,
+                val: anytype,
+                options: ValueOptions,
+            ) Writer.Error!void {
+                try self.container.fieldArbitraryDepth(name, val, options);
+            }
+
+            /// Print a field prefix. This prints any necessary commas, the field name (escaped if
+            /// necessary) and whitespace as configured. Useful if you want to serialize the field
+            /// value yourself.
+            pub fn fieldPrefix(self: *Struct, name: []const u8) Writer.Error!void {
+                try self.container.fieldPrefix(name);
+            }
+        };
+
+        /// Writes ZON slices field by field.
+        pub const Slice = struct {
+            container: Container,
+
+            fn start(parent: *Self, options: SerializeContainerOptions) Writer.Error!Slice {
+                try parent.writer.writeByte('&');
+                return .{
+                    .container = try Container.start(parent, .anon, options),
+                };
+            }
+
+            /// Finishes serializing the slice.
+            ///
+            /// Prints a trailing comma as configured when appropriate, and the closing bracket.
+            pub fn finish(self: *Slice) Writer.Error!void {
+                try self.container.finish();
+                self.* = undefined;
+            }
+
+            /// Serialize an item. Equivalent to calling `itemPrefix` followed by `value`.
+            pub fn item(
+                self: *Slice,
+                val: anytype,
+                options: ValueOptions,
+            ) Writer.Error!void {
+                try self.container.field(null, val, options);
+            }
+
+            /// Serialize an item. Equivalent to calling `itemPrefix` followed by `valueMaxDepth`.
+            pub fn itemMaxDepth(
+                self: *Slice,
+                val: anytype,
+                options: ValueOptions,
+                depth: usize,
+            ) (Writer.Error || error{ExceededMaxDepth})!void {
+                try self.container.fieldMaxDepth(null, val, options, depth);
+            }
+
+            /// Serialize an item. Equivalent to calling `itemPrefix` followed by
+            /// `valueArbitraryDepth`.
+            pub fn itemArbitraryDepth(
+                self: *Slice,
+                val: anytype,
+                options: ValueOptions,
+            ) Writer.Error!void {
+                try self.container.fieldArbitraryDepth(null, val, options);
+            }
+
+            /// Print a field prefix. This prints any necessary commas, and whitespace as
+            /// configured. Useful if you want to serialize the item value yourself.
+            pub fn itemPrefix(self: *Slice) Writer.Error!void {
+                try self.container.fieldPrefix(null);
+            }
+        };
```

Comment:
ZON no longer has `&`, so this type and all related functions must be removed.

Reply:
Whoops good catch, fixed.